README.md

```diff
@@ -18,40 +18,99 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True - Due to the == not looking for an exact similarity, but a 'truthiness'.
+Console says: True
+
 **b)** `"1" === 1`
 
+False - 3 ==='s are looking for a 100% match, and string 1 is not exactly the same as number 1.
+Console says: False
+
 **c)** `x == 'x'`
 
+True - Again, because double ='s are looking to see a close fit, rather than an exact match, and x is similar to string x. Moreover, javascript will turn x into a string in an effort to 'help out'
+Console says: Uncaught (x is not defined)
+After defining x as 1, console says: false, my assumption being that variable x is not == string x.
+
 **d)** `x == (x+'')`
 
+True - Because '' is considered an empty string and thus nothing, leaving x to be evaluated to itself.
+Console says: True (x defined as 1, otherwise says undefined)
+
 **e)** `'' == ' '`
 
+False - In this case '' is an empty string, whereas ' ' actually contains content, in this case a space.
+Console says: False
+
+
 **f)** `x = true`
 
+True - Because you are declaring x as true.
+Console says: True
+
 **g)** `var x; x == 'undefined'`
```

Comment:
1g) Assuming that x does not have any previous values, its value here is `undefined`, which is not equal to (and does not autoconvert to) `"undefined"`.